go/Dockerfile

```diff
@@ -1,16 +1,16 @@
 FROM ubuntu:14.04
 MAINTAINER Keybase <admin@keybase.io>
 
-VOLUME /keybase/bin
-VOLUME /keybase/socket
-
-ADD test/start_daemon_in_container.sh /start_daemon.sh
+RUN apt-get update && apt-get install -y fuse
 
 RUN useradd -m keybase
+RUN mkdir /keybase
+RUN chown keybase:keybase /keybase
 USER keybase
-ENV KEYBASE_RUN_MODE devel
-ENV KEYBASE_SOCKET_FILE /keybase/socket/keybase.sock
-ENV KEYBASE_DEBUG true
-ENV PATH=$PATH:/keybase/bin
+WORKDIR /home/keybase
 
-CMD /start_daemon.sh
+ENV KEYBASE_DEBUG=0
```

Comment:
It's so we can do stuff like this without explicitly stating the debug variable: 
`$ docker exec 47940d7a9b0a8a704addf0509aae3611592b57d5132e2073f9038a1e3a56c730 sh -c keybase status 2>/dev/null | grep -A3 Device: | grep name: | awk '{print $2}'`

[The services](https://github.com/keybase/kbfs/blob/1fa6627b25936c28b72d8a238f14eecbf071ae75/test/keybase_docker_setup.sh#L6) are still called with `KEYBASE_DEBUG=1`:
`KEYBASE_DEBUG=1 keybase service 2>&1 | tee -a service.log >/dev/null &`